android/app/build.gradle

```diff
@@ -36,10 +36,13 @@ def testAppDir = file("$projectDir/../../")
 apply from: file("${testAppDir}/test-app.gradle")
 applyTestAppModule(project, "com.react.testapp")
 
+project.ext.isRntaProject = projectDir.getParent() != null &&
+    (rootDir == file(projectDir.getParent()))
+
 project.ext.react = [
     applicationId: getApplicationId(),
     enableFlipper: getFlipperVersion(rootDir),
-    enableHermes: true,
+    enableHermes : true,
```

Comment:
Revert?

Reply:
This is the result of running the AS formatter. It aligns columns using the `:` char as delimeter.